src/pymor/discretizers/builtin/cg.py

```diff
@@ -1202,6 +1227,17 @@ def discretize_stationary_cg(analytical_problem, diameter=None, domain_discretiz
 
     data = {'grid': grid, 'boundary_info': boundary_info}
 
+    if energy_product:
+        energy_product = eL.parse_parameter(energy_product)
+        if preassemble:
+            eL = eL.assemble(energy_product)
+        else:
+            from pymor.operators.constructions import FixedParameterOperator
+            eL = FixedParameterOperator(eL, mu=energy_product)
+        products = {**m.products}
+        products['energy'] = 0.5*eL + 0.5*eL.H
```

Comment:
Shouldn't the symmetrization be unnecessary?

Reply:
Not for non-symmetric diffusion tensors (not sure if we support these anyway). If not, I guess it is ...

Reply:
Oh, didn't think about this possibility. Then I would propose to only symmetrize in case `p.diffusion` has non-scalar `shape_range`.

Reply:
Not sure how to reliable check for scalar numbers, but I tried something (`isinstance(..., Number)` misses scalar arrays and `np.isscalar(...)` allows too much).